UsersGuide/source/omjulia.rst

```diff
@@ -19,55 +19,228 @@ ZeroMQ - high performance asynchronous messaging library and it supports the Mod
 Standard Library version 3.2 that is included in starting with
 OpenModelica 1.9.2.
 
-The Development is in progress and will be soon available. At the moment
-a prototype of OMJulia is completed where the users will be able to import the
-OMJulia package in Julia and start to interact with omc through API calls
+To install OMJulia follow the instructions at https://github.com/OpenModelica/OMJulia.jl
 
-Current Prototype
-~~~~~~~~~~~~~~~~~
-The current version of the OMJulia package contains the following features:
+Features of OMJulia
+~~~~~~~~~~~~~~~~~~~
+The OMJulia package contains the following features:
 
-- Import the OMJulia package
+- Interactive session handling, parsing, interpretation of commands and
+  Modelica expressions for evaluation, simulation, plotting, etc.
 - Connect with the OpenModelica compiler through zmq sockets
 - Able to interact with the OpenModelica compiler through the :ref:`available API <scripting-api>`
+- Easy access to the Modelica Standard library.
 - All the API calls are communicated with the help of the sendExpression method implemented in a Julia module
 - The results are returned as strings
 
 Test Commands
 ~~~~~~~~~~~~~
 
-To get started, create a OMJulia session object:
+To get started, create a omc OMJulia session object:
 
 >>> using OMJulia
->>> omc= OMJulia()
+>>> omc= OMJulia.OMCSession()
 >>> omc.sendExpression("getVersion()")
-'v1.13.0-dev-531-gde26b558a (64-bit)'
+"v1.13.0-dev-531-gde26b558a (64-bit)"
```

Comment:
Shouldn't this ask OMPython for the version? We shouldn't have other versions hardcoded in the docs

Reply:
This is OMJulia, i think the docs does not use julia to compute anything, May be i can remove the getVersion()